params.h

```diff
@@ -9,7 +12,7 @@
     #include <arpa/inet.h>
 #endif
 
-#if defined(__linux__) || defined(_WIN32)
+#if defined(__linux__) || defined(__NetBSD__) || defined(_WIN32)
```

Comment:
Yes, i am still trying to implement sending fakes so keep it defined. If i won't get it working i remove this.

Reply:
> timeout is not supported in BSD

There are socket options `SO_RCVTIMEO` and `SO_SNDTIMEO` which are part of posix and can set timeout for sockets on most unix systems. Or they won't work because of non-blocking sockets?

Reply:
> Or they won't work because of non-blocking sockets?

won't